parlai/agents/tfidf_retriever/build_db.py

```diff
@@ -10,20 +10,19 @@
 
 import sqlite3
 import os
-import logging
+import logging  # Used only for constants
```

Comment:
perhaps we can make these class constants within `ParlaiLogger`, or perhaps just redefine them in `parlai.core.logging_utils` as module constants, so you could do something like this:

`from parlai.core.logging_utils import ParlaiLogger, INFO_LEVEL, ...`